### PR TITLE
OPR-GH-1294: n4jsd additions for next

### PR DESCRIPTION
**Pull request by dbo**

### CODE REVIEW

File: npm/next/5.0.0/config.n4jsd [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default external public function getConfig(): ~Object with {
+    serverRuntimeConfig: Object;
+    publicRuntimeConfig: Object;
+};
+
+/* -*- mode: typescript;-*- */
```
Comment (yoosiba) [inline npm/next/5.0.0/config.n4jsd]: repo has colouring configured, what is the point of copy&pasting this comment?
https://github.com/NumberFour/n4jsd/blob/master/.gitattributes#L1

Comment (dbo) [reply npm/next/5.0.0/config.n4jsd]: there is no point, was just taking an existing n4jsd file as a blueprint...

**Comment by mmews-n4** [comment]
@dbo what about this PR?

**Comment by dbo** [comment]
@mmews-n4 can you please have a look whether this conform to the new n4jsd publishing?

**Review by mmews-n4 — APPROVED** [review]
👍

**Pull request merged**